### PR TITLE
Changes to query log docs w.r.t. GQL codes

### PR DESCRIPTION
**Pull request by WilcoNeo**

### CODE REVIEW

File: modules/ROOT/pages/monitoring/logging.adoc [hunk 1]
```diff
@@ -1292,6 +1292,11 @@ The query log includes the GQL error information under the JSON object `errorInf
 * `position` -- The position (a JSON object containing a field for `column`, `offset`, and `line`) in the query where this error occurred.
 * `cause` -- A JSON object containing the `errorInfo` JSON object of the cause of the current `errorInfo` JSON object.
 
+GQLSTATUS code `50N42` is our default code. This code is shown when the exception does not have a GQL object yet.
```
Comment (renetapopova) [inline modules/ROOT/pages/monitoring/logging.adoc:1295]: How about this:
```suggestion
[NOTE]
====
The default GQLSTATUS code `50N42` is returned when an exception does not have a GQL object. 
Starting from Neo4j 5.25, GQL objects are added to exceptions; therefore, you can expect many `50N42` codes. 
However, it's important not to rely on this default code, as future Neo4j versions might change it by adding an appropriate GQL object to the exception. 
Additionally, GQL codes for external procedures are not yet stable.
====
```

**Review by renetapopova — CHANGES_REQUESTED** [review]
Thanks, @WilcoNeo. I added a suggestion.

File: modules/ROOT/pages/monitoring/logging.adoc (hunk 1 above)
Comment (WilcoNeo) [reply modules/ROOT/pages/monitoring/logging.adoc:1295]: Good :) Thanks!

File: modules/ROOT/pages/monitoring/logging.adoc [hunk 2]
```diff
@@ -1292,6 +1292,17 @@ The query log includes the GQL error information under the JSON object `errorInf
...
+[NOTE]
+====
+The default GQLSTATUS code `50N42` is returned when an exception does not have a GQL object. 
+Starting from Neo4j 5.25, GQL objects are added to exceptions; therefore, you can expect many `50N42` codes. 
+However, it's important not to rely on this default code, as future Neo4j versions might change it by adding an appropriate GQL object to the exception. 
+Additionally, GQL codes for external procedures are not yet stable.
+====
+Neo4j version 5.25 started to add GQL objects to exceptions. Therefore, many `50N42` codes can be expected.
+Note: You should not rely on this default code, since future Neo4j versions might add an appropriate GQL object to the exception (therefore changing this code into another one).
+Also, GQL codes for external procedures are not stable yet.
```
Comment (WilcoNeo) [inline modules/ROOT/pages/monitoring/logging.adoc:1304]: These 3 lines can then be removed right?

Comment (renetapopova) [reply modules/ROOT/pages/monitoring/logging.adoc:1304]: yes, thank you so much

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1865.surge.sh

Updated pages:
- [Logging](https://neo4j-docs-operations-1865.surge.sh/operations-manual/5/monitoring/logging)

**Review by renetapopova — APPROVED** [review]
Thanks.

**Comment by WilcoNeo** [comment]
[Documentation for query log changes 5.x](https://trello.com/c/4rL6vP6l/302-documentation-for-query-log-changes-5x)

**Pull request merged**